android/app/src/main/java/org/mozilla/magnet/permissions/Check.java

```diff
@@ -0,0 +1,55 @@
+package org.mozilla.magnet.permissions;
+
+import android.app.Activity;
+import android.app.AlertDialog;
+import android.app.Dialog;
+import android.content.DialogInterface;
+import android.content.pm.PackageManager;
+import android.support.v4.content.ContextCompat;
+
+/**
+ * Created by wilsonpage on 09/11/2016.
+ */
+abstract class Check {
```

Comment:
FYI Android studio told me to remove unnecessary scope declarations when something is `Package local`.